CMakeLists.txt

```diff
@@ -106,8 +103,7 @@ set (HEADERS
     inc/UDPSocket.h
     inc/TCPServer.h
     inc/BatteryModel.h
-    inc/CommManager.h
-    inc/Communicator.h
+        inc/Communicator.h
```

Comment:
The indentation doesn’t match here. Are you mixing tabs and spaces?

Reply:
Same problem in all the CMAKELISTS

Reply:
When I delete files, CLion automatically modifies the CMakeLists. I'll manually fix this.